### PR TITLE
MTDSA:21659 - Copy over shared code

### PR DESCRIPTION
**Pull request by DavouJ**

### CODE REVIEW

File: resources/public/api/conf/3.0/application.yaml [hunk 1]
```diff
@@ -53,31 +54,25 @@ tags:
...
   - name: Coding Out Underpayments and Debts
     description: Coding out is a concept used to retrieve underpaid tax and debts through a taxpayers PAYE tax code.<br><br> The coding out underpayments and debts endpoints allow software packages to overwrite coding out underpayment and debt amounts that are held by HMRC for a previous tax year and given NINO. Software packages can then amend previously submitted overwrite amounts, retrieve the HMRC held or user submitted coding out amounts or delete the user submitted amounts.<br><br>
-# {{#if (enabled 'cl402')}}
```
Comment (EdCope) [inline resources/public/api/conf/3.0/application.yaml:56]: These comments are required for rendering in OAS

File: project/CodeCoverageSettings.scala [hunk 2]
```diff
@@ -29,7 +29,8 @@ object CodeCoverageSettings {
     ".*Routes.*",
     "config.*",
     "testOnly.*",
-    "testOnlyDoNotUseInAppConf.*"
+    "testOnlyDoNotUseInAppConf.*",
+    "shared.config.*"
```
Comment (EdCope) [inline project/CodeCoverageSettings.scala:33]: Locally I got over 95% for coverage for this branch, can check this again when fully migrated if you prefer

**Review by EdCope — CHANGES_REQUESTED** [review]
Theres a small update to the shared code since you included it, might be worth adding in this PR or the implementation

File: resources/public/api/conf/3.0/application.yaml (hunk 1 above)
Comment (DavouJ) [reply resources/public/api/conf/3.0/application.yaml:56]: Didn't realise you were working on this also. Thought I added those lines by mistake.

**Review by EdCope — APPROVED** [review]

**Pull request merged**